docs/how-to/rocm-for-hpc/index.rst

```diff
@@ -0,0 +1,239 @@
+.. meta::
+   :description: How to use ROCm for HPC
+   :keywords: ROCm, AI, high performance computing, HPC, usage, tutorial
```

Comment:
not sure about:
usage
tutorial

Actually, most of these look generic. 
Perhaps add the name of each HPC application.